dom-querying/main.js

```diff
@@ -0,0 +1,19 @@
+console.log('hello, world');
+
+const h1 = document.querySelector('h1');
+console.log(h1);
+console.dir(h1);
+
+const explanation = document.querySelector('#explanation');
+console.log(explanation);
+console.dir(explanation);
+
+const hint = document.querySelector('.hint');
+console.log(hint);
+console.dir(hint);
+
+const p = document.querySelectorAll('p');
+console.log(p);
+
+const exampleLink = document.querySelectorAll('.example-link');
+console.log(exampleLink);
```

Comment:
Don't forget to include labels when using `console.log`! It's a good habit you really want to build early

`console.log('h1', h1)`